programs/sbf/rust/sysvar/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub fn process_instruction(
         sysvar::clock::id().log();
         let clock = Clock::from_account_info(&accounts[2]).unwrap();
         assert_ne!(clock, Clock::default());
-        let got_clock = Clock::get()?;
+        let got_clock = Clock::get().unwrap();
```

Comment:
Yes looks funny I know. All of these `?` => `.unwrap()` changes are in the commit [Workaround for rustc crash caused by 16 byte alined memcpy.](https://github.com/anza-xyz/agave/pull/1711/commits/5b39e3c0d804f2fc5c16a91e66b9da031566702b)

Lucas is already working on fixing that, but it takes one release cycle of the platform tools, so this is the workaround for now.

Reply:
How does this workaround actually work, though?

Reply:
The memcpy is not explicit it happens because of stack spilling / register pressure as far as I can tell. So pretty much any change to the code can cause it to go either way. With the next release of the platform tools that should be fixed, and until then the behavior is stable.